test/terraform_basic_example_test.go

```diff
@@ -1,26 +1,35 @@
 package test
 
 import (
+	"path/filepath"
 	"testing"
 
+	"github.com/gruntwork-io/terratest/modules/files"
+
 	"github.com/gruntwork-io/terratest/modules/terraform"
 	"github.com/stretchr/testify/assert"
 )
 
 // An example of how to test the simple Terraform module in examples/terraform-basic-example using Terratest.
 func TestTerraformBasicExample(t *testing.T) {
 	t.Parallel()
+	terraformDir := "../examples/terraform-basic-example"
 
 	expectedText := "foo"
 
 	terraformOptions := &terraform.Options{
 		// The path to where our Terraform code is located
-		TerraformDir: "../examples/terraform-basic-example",
+		TerraformDir: terraformDir,
 
 		// Variables to pass to our Terraform code using -var options
 		Vars: map[string]interface{}{
 			"example": expectedText,
 		},
+		// Only create the example1 file
+		Targets: []string{"local_file.example"},
+
+		// Use the var files
+		VarFiles: []string{"varfile.tfvars"},
```

Comment:
Hm, `terraform_basic_example_test.go` should be, well, a basic example intended to provide simple example code for Terratest newbies. I think `Targets` and `VarFiles` and the corresponding checks may be a little too complex for someone just starting out. Perhaps move this example + corresponding test code to a different example?